app/src/main/java/com/mamadou/newsapp/models/Article.kt

```diff
@@ -1,16 +1,17 @@
 package com.mamadou.newsapp.models
 
 import android.os.Parcelable
+import com.squareup.moshi.Json
 import kotlinx.parcelize.Parcelize
 
 @Parcelize
 data class Article(
-    val author: String? = null,
-    val title: String,
-    val description: String,
-    val url: String? = null,
-    val urlToImage: String,
-    val publishedAt: String,
-    val content: String? = null,
-    val source: Source
+    @field:Json(name = "author") val author: String? = null,
+    @field:Json(name = "title") val title: String,
+    @field:Json(name = "description") val description: String,
+    @field:Json(name = "url") val url: String? = null,
+    @field:Json(name = "urlToImage") val urlToImage: String,
+    @field:Json(name = "publishedAt") val publishedAt: String,
+    @field:Json(name = "content") val content: String? = null,
+    @field:Json(name = "source") val source: Source,
```

Comment:
Great job adding the annotations! For Gson and Moshi converters, you can also remove these annotations in this case since the JSON property names match identically with the variable names.  Typically you'd only need the @Json annotation if the variable names are different.